arm/Microsoft.OperationalInsights/workspaces/.parameters/parameters.json

```diff
@@ -32,14 +32,14 @@
             "value": [
                 {
                     "name": "Automation",
-                    "resourceId": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Automation/automationAccounts/sxx-wd-aut-x-001"
+                    "resourceId": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Automation/automationAccounts/adp-sxx-wd-aut-x-001"
                 }
             ]
         },
         "savedSearches": {
             "value": [
                 {
-                    "name": "VMSSQueries3",
+                    "name": "VMSSQueries4",
```

Comment:
Does this need to change at every deployment? Asking as in this case we may end up having a failure for next runs

Reply:
Yes, if not updated then it throws and error saying there is updated customer information. Check the error here:
![image](https://user-images.githubusercontent.com/37813342/141820613-15098b2d-0e1a-412f-881d-0beaf1e11d09.png)